config/opensearch_dashboards.yml

```diff
@@ -160,4 +160,8 @@
     # defaultUrl: ""
     # darkModeUrl: ""
   # faviconUrl: ""
-  # applicationTitle: ""
\ No newline at end of file
+  # applicationTitle: ""
+
+# Set the value of this setting to true to capture region blocked warnings and errors 
+# for your map rendering services.
+# map.captureRegionBlockedError: false
```

Comment:
Can we be more specific about what "capture" means in this context? `map.showRegionBlockedWarning` might be a better name, but that should be open to discussion.